OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -266,13 +266,17 @@ public void UpdateMouseover()
 				return;
 			}
 
-			if (resourceRenderer != null)
+			if (resourceRenderers.Any())
```

Comment:
You can remove this `if` now I think, since if there aren't any elements the for loop will simply run 0 times.

Reply:
You are right. Fixed.